pandas/_libs/lib.pyx

```diff
@@ -538,6 +538,24 @@ def has_infs(floating[:] arr) -> bool:
     return ret
 
 
+@cython.wraparound(False)
+@cython.boundscheck(False)
+def has_NA(ndarray arr) -> bool:
+    """
+    Return True if NA present in arr, False otherwise
+    """
+    cdef:
+        Py_ssize_t i
+
+    assert arr.ndim == 1, "'arr' must be 1-D."
+
+    for i in range(len(arr)):
+        if arr[i] is C_NA:
+            return True
+
+    return False
```

Comment:
This may be overkill, but fits with some other cython methods like `has_infs` and would hopefully be useful in other locations

Reply:
could live in _libs.missing?

Reply:
have moved